multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1152,7 +1119,7 @@ def _fit(
         )
 
         config = update_config_by_rules(
-            problem_type=self._problem_type,
+            problem_type=self.problem_type,
```

Comment:
I see all `self._problem_type` is replaced by `self.problem_type`. Mind to tell the reasons?

Reply:
I can also change all self.problem_type to self._problem_type so that it will be consistent.

Reply:
One advantage of `self.problem_type` over `self._problem_type` is that you can never use `self.problem_type = XXX` which accidentally overwrite the value.